### ISSUE TITLE
Escape Game Fort Boyard - CUSA19411

### ISSUE THREAD

**Issue by GAMERRPCS3** [red-prig/fpps4-game-compatibility#1596]
### Game Name

Escape Game Fort Boyard

### Game code

CUSA19411

### Game version

01.00

### Used emulator's version

b158919

### Current status

Boots

### Error

An unhandled exception occurred at $00000001000995A0:
EAssertionFailed: TODO tiling_idx:Thick_1dThick (vImageTiling.pas, line 499)

### Description

the game starts, and there is sound. then after some minute after loading it crashe.

### Used hacks

IMAGE_TEST_HACK

### Screenshots

![1](https://user-images.githubusercontent.com/121258975/234472501-03b2211f-a954-4f5d-93ba-814a6b53a3cf.png)
![2](https://user-images.githubusercontent.com/121258975/234472513-8349e87e-6dc1-49f1-8cde-1bb6564e63e2.png)

**Comment by gandalfthewhite19890404** [comment]
For TODO tiling_idx:Thick asserts use UNKNOW_TILING hack
retest

**Comment by gandalfthewhite19890404** [comment]
Also for what IMAGE_TEST_HACK?
DO NOT USE any hacks, except if they improve emulation in particular game.

**Comment by GAMERRPCS3** [comment]
> Also for what IMAGE_TEST_HACK?
> DO NOT USE any hacks, except if they improve emulation in particular game.

ok, i will test it again.

**Comment by GAMERRPCS3** [comment]
> For TODO tiling_idx:Thick asserts use UNKNOW_TILING hack retest

yes indeed, I rushed into the test. and I even forget to apply the game update to see if the results are different.

**Comment by dev-channe** [comment]
Retest
crashes before accessing the menu.
Used hacks - DISABLE_DEPTH_HACK - SKIP_UNKNOW_TILING
Game version - 01.02
Used emulator's version - b158919

![2](https://user-images.githubusercontent.com/121258975/234530594-16ffc63c-bee4-42a7-820f-a5fbce49fe0c.png)
![3](https://user-images.githubusercontent.com/121258975/234530607-90ea7a82-2598-4343-8548-d68771785ee7.png)
